masu/config.py

```diff
@@ -37,6 +37,15 @@ class Config(object):
     DB_HOST = os.getenv('DATABASE_HOST', 'localhost')
     DB_PORT = os.getenv('DATABASE_PORT', 15432)
 
+    # Koku Connectivity
+    KOKU_HOST = os.getenv('KOKU_HOST_ADDRESS', 'localhost')
+    KOKU_PORT = os.getenv('KOKU_PORT', 8000)
+    KOKU_ADMIN_USER = os.getenv('KOKU_ADMIN_USEER', 'admin')
```

Comment:
Do we want to stay consistent with the koku openshift template and use `SERVICE_ADMIN_USER`, `SERVICE_ADMIN_PASSWORD`?

Reply:
Since the two services could presumably live on separate hosts I think these environment variables should not be dependent on each other.   But I might as well make them the same just for a simplified local environment where masu and koku are running on the same host.